src/apps/store/dtos/book/search.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsOptional } from 'class-validator';
+
+export class BookSearchDto {
+  @IsOptional()
+  title?: string;
+}
```

Comment:
This class must extends `PaginateDto` to utilize `limit, skip, sort` query parameters.